templates/simple/package.json

```
@@ -4,7 +4,7 @@
     "kubernetes-models": "^0.8.1"
   },
   "devDependencies": {
-    "@socialgouv/kosko-charts": "^2.6.4",
+    "@socialgouv/kosko-charts": "link:../..",
```

We should keep the version on the template

```suggestion
    "@socialgouv/kosko-charts": "^2.6.4",
```